### PR TITLE
docs: describe project structure

### PR DESCRIPTION
**Pull request by mkucmus**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ You may want to read:
...
+
+#### There are 7 packages which that are part of the project.
+
+They are located in ./packages and can be developing and published separatelly.
+
+1. **cli** [_@shopware-pwa/cli_](https://www.npmjs.com/package/@shopware-pwa/cli) - A CLI tool for creating an instance of Shopware PWA projects.
+2. **commons** [_@shopware-pwa/commons_](https://www.npmjs.com/package/@shopware-pwa/commons) - A collection of all necessary interfaces shared across the project's ecosystem.
+3. **composables** [_@shopware-pwa/composables_](https://www.npmjs.com/package/@shopware-pwa/composables) - Vue 3 composables dedicated for Shopware. Composition API based.
+4. **default-theme** [_@shopware-pwa/default-theme_](https://www.npmjs.com/package/@shopware-pwa/default-theme) - Default theme for Shopware PWA. Nuxt based.
+5. **helpers** [_@shopware-pwa/helpers_](https://www.npmjs.com/package/@shopware-pwa/helpers) - Functions that transform/convert Shopware6 specific data that comes from the API. Widely used by _composables_ or _default-theme_.
```
Comment (Michal-Dziedzinski) [inline CONTRIBUTING.md:39]:
```suggestion
5. **helpers** [_@shopware-pwa/helpers_](https://www.npmjs.com/package/@shopware-pwa/helpers) - Functions that transform/convert Shopware 6 specific data that comes from the API. Widely used by _composables_ or _default-theme_.
```

File: STRUCTURE.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Structure
+Short description of project's structure.
+
+#### There are 7 packages which that are part of the project.
```
Comment (Michal-Dziedzinski) [inline STRUCTURE.md:4]: There should probably be 7 packages described, shouldn't they?

Comment (mkucmus) [reply STRUCTURE.md:4]: :+1:

**Comment by patzick** [comment]
closes #581

**Pull request merged**